tests/data_files/Makefile

```diff
@@ -315,6 +315,15 @@ rsa_pkcs8_2048_public.der: rsa_pkcs8_2048_public.pem
 	$(OPENSSL) rsa -pubin -in $< -outform DER -pubout -out $@
 all_final += rsa_pkcs8_2048_public.der
 
+authorityKeyId_subjectKeyId.crt: 
```

Comment:
There is an extraneous space at the end of this line.

Reply:
Thanks for the finding, I have just corrected it